go/enclave/components/batch_executor.go

```diff
@@ -444,7 +447,7 @@ func (executor *batchExecutor) processTransactions(
 		} else {
 			// Exclude all errors
 			excludedTransactions = append(excludedTransactions, tx.Tx)
-			executor.logger.Info("Excluding transaction from batch", log.TxKey, tx.Tx.Hash(), log.BatchHashKey, batch.Hash(), "cause", result)
+			executor.logger.Debug("Excluding transaction from batch", log.TxKey, tx.Tx.Hash(), log.BatchHashKey, batch.Hash(), "cause", result)
```

Comment:
this is unrelated, but it has to be done because this entry is too verbose

Reply:
Maybe we need to prioritise the issues that leaves tx stuck in the mempool forever and then re-enable this. Because I think this failure has no feedback if it's got this far, so user will just see spinning forever waiting for receipt and this sometimes gave us visibility on them.